backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/irasdescriptors/FakeIrUtils.kt

```diff
@@ -25,15 +26,10 @@ internal val IrDeclaration.module get() = this.descriptor.module
 internal val IrField.isDelegate get() = @Suppress("DEPRECATION") this.descriptor.isDelegated
 
 internal fun IrFunction.getObjCMethodInfo() = this.descriptor.getObjCMethodInfo()
-internal fun IrClass.isExternalObjCClass() = this.descriptor.isExternalObjCClass()
-internal fun IrClass.isKotlinObjCClass() = this.descriptor.isKotlinObjCClass()
 internal fun IrFunction.getExternalObjCMethodInfo() = this.descriptor.getExternalObjCMethodInfo()
 internal fun IrFunction.isObjCClassMethod() = this.descriptor.isObjCClassMethod()
-internal fun IrFunction.canObjCClassMethodBeCalledVirtually(overridden: IrFunction) =
-        this.descriptor.canObjCClassMethodBeCalledVirtually(overridden.descriptor)
-internal fun IrClass.isObjCClass() = this.descriptor.isObjCClass()
+
```

Comment:
I found lot of such changes, shouldn't we keep commit minimal?

Reply:
These are not just some random removals. Those several functions were rewritten to not use descriptors anymore (see below in the same file). That was required to please lots of new wrapped descriptors coming out of the ir deserializer.